packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -765,6 +790,10 @@ const DatePickerInput = (props) => {
     { 'is-invalid': isInvalid },
   ]);
 
+  // Indicates selected date from calendar popup for SR
+  const calendarDate = DateUtil.isValidDate(value, momentDateFormat) ? `${getLocalizedDateForScreenReader(DateUtil.createSafeDate(dateValue, initialTimeZone), { intl, locale: intl.locale })} ${intl.formatMessage({ id: 'Terra.datePicker.selected' })}` : label;
+  const inputDate = DateUtil.isValidDate(value, momentDateFormat) ? `${getLocalizedDateForScreenReader(DateUtil.createSafeDate(dateValue, initialTimeZone), { intl, locale: intl.locale })}` : '';
```

Comment:
what is difference between calendar date and input date.

Reply:
calendarDate is used to indicate the selected date in calendar popup (will be passed as part of aria-label for button)
 calendarDate value would be like "<date> selected"
 inputDate has the entered date (if valid) and it was added to convey the date along with format. It provides context when hot keys are used as well. It's value is only the date.

Reply:
refactored in https://github.com/cerner/terra-framework/pull/1774/commits/67e9043fe4a041fcc8573ca4f0cb82cc709ae204